### DISCUSSION TITLE
好无聊！！！

### DISCUSSION THREAD

**Discussion by mengbaijiu** [share121/inter-knot#941] [其他]
好无聊好无聊好无聊好无聊，没什么游戏能玩农被打爆了，京不想玩，绝在长草，瓦在等朋友 
![7591AA5343AF6FE03D2C8A948D1F4340](https://github.com/user-attachments/assets/48cc0898-52b8-485f-978f-27f1d7fd7a93)

**Comment by capacitor1** [comment]
我推荐一款开放世界大型游戏：





Minecraft
![image](https://github.com/user-attachments/assets/05a8bdb6-9886-4597-9641-2b01e0586677)

**Reply by mengbaijiu** [reply]
mc啊也感觉没什么好玩的，虽然一玩就可能几个小时鹅鹅鹅

**Comment by warning-Kirov-reporting** [comment]
原始人，启动！

**Reply by mengbaijiu** [reply]
原始人，启动！

**Comment by 3437798024** [comment]
推荐一款，克鲁赛德战记，虽然我玩的也不多不过我觉得打发时间应该不错。

**Comment by hzjwork** [comment]
我一般无聊就玩自动化游戏，异星工厂，戴森球计划什么的，一玩起来不知不觉就是一通宵

**Comment by ShallowDream12321** [comment]
永劫无间

**Comment by GanlinSama** [comment]
回一楼，MC是沙盒游戏，也能算是开放世界么？

**Comment by ShallowDream12321** [comment]
@GanlinSama 应该不算 我国际版Java1.20.4的
